sefkhet-abwy/overlays/cnv-prod/new-label-normaliyer.yaml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
please fix the name of the file `new-label-normalizer` , it would cause confusion for us later.

Reply:
:)